### PR TITLE
Add attribute-based cleanup commands.

### PR DESCRIPTION
**Pull request by rcaballeromx**
Signed-off-by: rcaballeromx <grca@google.com>

### CODE REVIEW

File: content/docs/tasks/policy-enforcement/denial-and-list/index.md [hunk 1]
```diff
@@ -245,7 +245,15 @@ subnet.
 
 ## Cleanup
 
-* Remove the mixer configuration:
+* Remove the mixer configuration for attribute-based white- and blacklisting:
```
Comment (geeknoid) [inline content/docs/tasks/policy-enforcement/denial-and-list/index.md]: Could you change all instances of "mixer" with "Mixer" in this doc? Mixer is a proper name...

Thanks.

Comment (rcaballeromx) [reply content/docs/tasks/policy-enforcement/denial-and-list/index.md]: Done! Thanks for catching them.

**Comment by rcaballeromx** [comment]
@geeknoid All fixed. PTAL

**Pull request merged**